pandas/tests/tseries/offsets/test_fiscal.py

```diff
@@ -604,3 +605,17 @@ def test_offset(self):
         assert_offset_equal(offset2,
                             datetime(2013, 1, 15),
                             datetime(2013, 3, 30))
+
+
+def test_bunched_yearends():
+    # GH#14774 cases with two fiscal year-ends in the same calendar-year
+    fy = FY5253(n=1, weekday=5, startingMonth=12, variation='nearest')
+    dt = Timestamp('2004-01-01')
+    assert fy.rollback(dt) == Timestamp('2002-12-28')
+    assert (-fy).apply(dt) == Timestamp('2002-12-28')
+    assert dt - fy == Timestamp('2002-12-28')
```

Comment:
can you add from the OP: ``fy.rollback(pd.Timestamp('2003-12-31'))`` (which I think is the same result) as well